sweetest/src/main/java/com/mysugr/sweetest/framework/base/BaseJUnitTest.kt

```diff
@@ -4,7 +4,12 @@ import com.mysugr.sweetest.framework.build.TestBuilder
 import com.mysugr.sweetest.framework.configuration.ModuleTestingConfiguration
 import org.junit.Before
 
-abstract class BaseJUnitTest(private val moduleTestingConfiguration: ModuleTestingConfiguration) : TestingAccessor {
+abstract class BaseJUnitTest @Deprecated(
```

Comment:
note to myself: i need to check how this deprecation displays in the IDE when using the lib...